apps/prairielearn/vitest.config.ts

```diff
@@ -75,7 +52,19 @@ const isRunningOnDist = process.argv
 
 export default defineConfig({
   test: {
-    reporters: process.env.GITHUB_ACTIONS ? ['default', new CustomGithubReporter()] : ['default'],
+    reporters: process.env.GITHUB_ACTIONS
+      ? [
+          'default',
+          [
+            'github-actions',
+            {
+              onWritePath(path: string) {
+                return path.replace(/^\/PrairieLearn\//, '');
```

Comment:
Nit: consider a comment explaining what's going on here, and specifically the fact that this is hardcoding the path in the container.

On that note: we could probably resolve the path relative to the repository root to avoid the hardcoding entirely.

Reply:
Added comment in [b3063dd](https://github.com/PrairieLearn/PrairieLearn/pull/12135/commits/b3063dd8a8749563090bbac03a0087778dcf8676). Can you explain how I would determine the path of the repository root? Just resolve 2 dirs up from this vitest config? I think the hardcoding is sufficient for now in any case.